Week1/homework/js-exercises/exercise7.js

```diff
@@ -0,0 +1,8 @@
+// exercise 7
+"use strict"
+let str="";
+function print(str) {
+    console.log(str);
+}
+let myStr="Programming is so interesting!";
+print("the lengt of the string is "+myStr.length);
```

Comment:
Also here, try to have some extra spacing for better readablity. 

So `"the lengt of the string is "+myStr.length` could be `"the lengt of the string is " + myStr.length`.

You could have also used the backticks (that you've done in other exercises). Like so:

```
print(`The lengt of the string is ${ myStr.length}`);
```